src/components/Dashboard.tsx

```diff
@@ -1,10 +1,48 @@
+import { useSelector } from 'react-redux'
+import { useFirestoreConnect } from 'react-redux-firebase'
+import { RootState } from '../store/store'
 import { Header } from './Header'
+import { ColumnList } from './ColumnList'
+import { ColumnForm } from './ColumnForm'
 
 export const Dashboard = () => {
+  const { uid } = useSelector((state: RootState) => state.firebase.auth)
+  useFirestoreConnect({
+    collection: `users/${uid}/columns`,
+    storeAs: 'columns',
+  })
+  const columns = useSelector(
+    (state: RootState) => state.firestore.data.columns
+  )
+  const columnsArray = Object.entries(columns ?? []).map(([id, column]) => ({
+    id,
+    ...column,
+  }))
```

Comment:
What do you think about renaming it to `columnsData`? And then you could use pure `columns` for the actual data you're using. The same could be applied to the `cards`.
```suggestion
  const columnsData = useSelector(
    (state: RootState) => state.firestore.data.columns
  )
  const columns = Object.entries(columnsData ?? []).map(([id, column]) => ({
    id,
    ...column,
  }))
```